setup.py

```diff
@@ -22,7 +22,7 @@
         'paramiko==2.4.2',
         'pyyaml>=4.2b1',
         'jinja2==2.10.1',
-        'kubernetes<9.0.0,>=8.0.0',  # to satisfy openshift package requirement
+        'kubernetes>=9.0.0',  # to satisfy openshift package requirement
```

Comment:
I think the comment here was to justify why we specified <9>=8. As long as this has been tested and works with 9.0.0 let's remove the comment as well.

Reply:
Yep, will remove

Reply:
@shreekarSS please update this PR so we can approve and merge, thanks!